.circleci/config.yml

```diff
@@ -194,6 +193,20 @@ jobs:
           name: "Link packages"
           command: |
             yarn install
+      - run:
+          name: "Build static React kitchen-sink"
```

Comment:
Let's not add it to `unit-test` job but rather replace current "integration tests" in `example-kitchen-sinks` with image snapshots

Reply:
Maybe we could even reuse some code between those two files:
https://github.com/thomasbertet/storybook/blob/9d0bfbf17d69e3954bb7d5701b5bd14a7ba1f405/addons/storyshots/src/test-body-image-snapshot.js
https://github.com/thomasbertet/storybook/blob/9d0bfbf17d69e3954bb7d5701b5bd14a7ba1f405/integration/examples.test.js

Reply:
Can this step be removed now?

Reply:
Ah no, it can't since the "integration" test are relying on it. 
I forgot to answer when I looked at it sorry for the delay :)

Reply:
This is `unit-test` job, integration test is in another place